recipe_templates/mpdaf/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+export PKG_CONFIG_PATH=$PREFIX/lib/pkgconfig
```

Comment:
Not sure what is the best way to be able to use the cfitsio package, this seems fine but maybe there is a better way ?